core/src/main/java/com/novoda/noplayer/Player.java

```diff
@@ -33,8 +33,18 @@
 
     void selectAudioTrack(PlayerAudioTrack audioTrack);
 
+    boolean hasAvailableSubtitles();
+
+    void selectSubtitleTrack(PlayerSubtitleTrack subtitleTrack, PlayerSubtitlesView playerSubtitlesView);
+
+    void selectFirstAvailableSubtitleTrack(PlayerSubtitlesView playerSubtitlesView);
```

Comment:
Do we have an equivalent method for `audioTrack`? If we don't how do we select the first available one for audio? We should push for symmetry